syntax = "proto3";
package notification;

option go_package = "/proto_files/notification"; //REPLACE WITH YOUR GITHUB PROJECT DIRECTORY PATH
import "google/protobuf/timestamp.proto";

message Notification {
  string id = 1;
  string telegram_handle = 2;
  string notification_type = 3;
  string content = 4;
  google.protobuf.Timestamp created_at = 5;
}

